main.go

```diff
@@ -76,7 +76,7 @@ func main() {
 		options.CtrlConfig.MaxConcurrentReconciles = 1
 	}
 
-	ctrl.SetLogger(zap.Logger(true))
+	ctrl.SetLogger(zap.New(zap.UseDevMode(true)))
```

Comment:
the old style `zap.Logger(true)` indicates a DevMode=true, however it may not that reasonable, and I've update it be false defaulted to only stacktraces on errors